pkg/loop/internal/pb/reporting_plugin_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb";
+
+package loop;
+
+service ReportingPluginService {
+  rpc NewReportingPluginFactory (NewReportingPluginFactoryRequest) returns (NewReportingPluginFactoryReply) {}
+}
+
+message ReportingPluginServiceConfig {
+  string providerType = 1;
+  string command = 2;
+  string pluginName = 3;
+  string pluginConfig = 4;
```

Comment:
i'm reticent to say it, since i see this was just changed do to a comment from @jmank88, but i think bytes is a better type here because it's the most neutral and this is low level wire communication.

Reply:
What do you mean by most neutral? Like we might use config that is not human readable in the future? Do you think we should change the Relayer TOML field from `string` to `bytes`?

Reply:
https://protobuf.dev/programming-guides/proto3/#scalar
IIUC they are the same cost and there is only a superficial difference of limiting strings to utf8.

Reply:
i mean that the plugin config is really a struct or structs. representing that as bytes feels better and more intuitive to me over the wire.

but i'm not going to die on that hill